Polyhedron/demo/Polyhedron/Scene_surface_mesh_item.h

```diff
@@ -0,0 +1,117 @@
+#ifndef CGAL_SCENE_SURFACE_MESH_ITEM_H
+#define CGAL_SCENE_SURFACE_MESH_ITEM_H
+//Defines the precision of the positions (for performance/precision sake)
+#define GL_DATA GL_FLOAT
+#define gl_data float
+#define IS_FLOAT 1
```

Comment:
In CGAL we have decided that macro names should start with `CGAL_`, to avoid naming conflicts with other libraries.